<?xml version="1.0"?>
<launch>

	<param name="use_sim_time" value="true"/>
	<node name="QLearning" pkg="grasp_learning" type="QLearning.py" output="screen">
		<param name="load_model" value="0" />
		<param name="model_name" value="model.ckpt" />
 		<param name="num_inputs" value="1" />
		<param name="num_outputs" value="1" />
		<param name="num_rewards" value="1" />
		<rosparam param="hidden_units">[20]</rosparam>
		<param name="learning_rate" value="0.01"/>
		<param name="batch_size" value="15" />
		<param name="discount_factor" value="0.99" />
		<param name="relative_path" value="$(find grasp_learning)/stored_data/tested_data/" />
		<param name="max_num_trials" value="5" />
		<param name="min_dist_norm" value="0.04" />
		<param name="epsilon" value="1" />
		<param name="num_actions" value="2" />
		<param name="num_samples" value="5000" />
 	</node>
</launch>
